recipes/yarn.rb

```diff
@@ -220,11 +220,21 @@
     dest "/user/#{node['hadoop_spark']['user']}/log4j.properties"
   end
 
-  bash 'materialize_truststore' do
+  encyption_password = "adminpw"
+  if node.attribute?('hopsworks') == true
+    if node['hopsworks'].attribute?('encyption_password') == true
+      encyption_password = node['hopsworks']['encyption_password']
+    end
+  end
+
+  bash 'materialize_truststore and convert to pem' do
     user "root"
     code <<-EOH
         cp -f #{node['kagent']['certs_dir']}/cacerts.jks /tmp
         chmod 755 /tmp/cacerts.jks
+        keytool -importkeystore -srckeystore cacerts.jks -destkeystore cacerts.p12 -srcstoretype jks -deststoretype pkcs12 -noprompt -srcstorepass #{encyption_password} -deststorepass #{encyption_password} 
+        openssl pkcs12 -in cacerts.p12 -out cacerts.pem -passin pass:#{encyption_password}
+        history -c
```

Comment:
Clearing the whole history might be too aggressive. Try `echo "YOUR_COMMAND && history -d $((HISTCMD-1))` instead